### PR TITLE
Tidy up nature-header at narrow screen widths

### PR DESCRIPTION
**Pull request by ricky-lau**
https://jira.springernature.com/browse/PAN-1068

### Before
www nature com_(iPhone 5_SE)

### After
local-www nature com_7890_nature(iPhone 5_SE) (3)

### CODE REVIEW

File: toolkits/nature/packages/nature-header/scss/50-components/header.scss [hunk 1]
```diff
@@ -53,11 +52,25 @@
...
+		}
+	}
+}
+
+.c-header__menu--journal {
 	font-weight: $font-weight-bold;
 }
 
-.c-header__menu--secondary {
+.c-header__menu--updates {
```
Comment (tulipdexter) [inline toolkits/nature/packages/nature-header/scss/50-components/header.scss:73]: What does 'updates' mean in the header?

Comment (ricky-lau) [reply toolkits/nature/packages/nature-header/scss/50-components/header.scss:73]: it was an attempt to group the sign up for alerts and rss feed links together

Comment (tulipdexter) [reply toolkits/nature/packages/nature-header/scss/50-components/header.scss:73]: I was trying to group that mean and the journal menu as the same 'thing'. That's why I went with secondary. Stylistically they're the same.

Comment (tulipdexter) [reply toolkits/nature/packages/nature-header/scss/50-components/header.scss:73]: But if you're keen to split them, maybe something like `--tools` (as we moved them from journal tools section

File: toolkits/nature/packages/nature-header/HISTORY.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 # History
 
+## 4.0.0 (2020-01-13)
```
Comment (tulipdexter) [inline toolkits/nature/packages/nature-header/HISTORY.md:3]: 2021 🎉 🍾 🎆

**Review by tulipdexter — APPROVED** [review]

**Pull request merged**